Docs: Person profiles updates (#8271)
PostHog/posthog.com: contents/docs/product-analytics/identify.mdx

> Note that when calling `alias` in the frontend SDKs, if you have set any properties onto the anonymous user, they will be merged into the user with `distinct_id`. For more details, see the FAQ on [how properties are managed when identifying anonymous users](/docs/data/identify#how-are-properties-managed-when-identifying-anonymous-users).

## Processing person profiles

> For user identification to work, you must be capturing person profiles. If set your `person_profile` config option is set to `identified_only`, PostHog only captures person profiles once you call one of the following functions or methods:

Reviewer: ```suggestion
If your `person_profile` config option is set to `identified_only`, PostHog creates a person profile once you call one of the following functions or methods:
```

- `identify`
- `alias`
- `setPersonProperties`
- `setPersonPropertiesForFlags`
- `group`
- `setGroupPropertiesForFlags`

Once you create a person profile for a user distinct ID, all events for that distinct ID count as ones with person profiles.

> 

Reviewer: ```suggestion

If your `person_profile` config option is set to `always`, PostHog creates a person profile for every user on your site, whether or not they have been officially identified.

```
